src/Plutarch/Extra/Value.hs

```diff
@@ -778,3 +781,46 @@ matchOrTryRec requiredAsset = phoistAcyclic $
       )
       (const def)
       # inputpvalue
+
+{- | Get the negative and positive amount of a particular 'CurrencySymbol', and
+     return nothing if it doesn't exist in the value.
+
+     @since 3.14.1
+-}
+psymbolValueOf' ::
```

Comment:
Looks good, but this is oddly specific :D

Reply:
Haha, it is. It's efficient for that specific use case, though.

Reply:
For context, see the discussion in https://github.com/Liqwid-Labs/agora/pull/202